app/domain/authentication/security/extract_request_remote_addr_info.rb

```diff
@@ -0,0 +1,51 @@
+require 'command_class'
+
+module Authentication
+  module Security
+    ExtractRequestRemoteAddrInfo = CommandClass.new(
+      dependencies: {
+        # InputValidation
+      },
+      inputs: [:request]
+    ) do
+
+      extend Forwardable
+      def_delegators :@request, :client_ip, :headers
```

Comment:
you should use here `remote_addr` instead of `client_ip`

Reply:
it was remote_addr when I added the delegate I switched by mistake